### PR TITLE
 modify proto files according the standard style

### PR DESCRIPTION
**Pull request by NoelBright**
### Proposed changes in this pull request
Explain the changes in this pull request in order to help the project maintainers understand the overall impact of it.

### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [ ] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [ ] Read the [CONTRIBUTION guidelines](https://github.com/nknorg/nkn#contributing).
- [ ] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [ ] Added proper documentation where ever applicable (in code and README.md).
- [ ] Code has been written according to [NKN-Golang-Style-Guide](https://github.com/nknorg/nkn/wiki/NKN-Golang-Style-Guide)

### Extra information
Any extra information related to this pull request.

### CODE REVIEW

File: pb/payload.proto [hunk 1]
```diff
@@ -28,44 +28,44 @@ enum PayloadType {
...
-	bytes Submitter    = 2;
+	bytes sigChain     = 1;
+	bytes submitter    = 2;
 }
 
 message DeleteName {
-	bytes  Registrant  = 1;
-    string Name        = 2;
+	bytes  registrant  = 1;
+    string name        = 2;
```
Comment (yilunzhang) [inline pb/payload.proto]: There seems to be some inconsistency here (tab vs space)

File: pb/block.proto [hunk 2]
```diff
@@ -7,20 +7,9 @@ import "pb/header.proto";
...
-//option (gogoproto.testgen_all) = true;
-//option (gogoproto.equal_all) = true;
-//option (gogoproto.populate_all) = true;
-
 
 message MsgBlock {
-	BlockHeader     Header                 = 1;
-    repeated MsgTx  Transactions           = 2;
+	BlockHeader     header                 = 1;
+    repeated MsgTx  transactions           = 2;
```
Comment (yilunzhang) [inline pb/block.proto]: Also formatting issue here

File: pb/payload.proto (hunk 1 above)
Comment (NoelBright) [reply pb/payload.proto]: done.

File: pb/block.proto (hunk 2 above)
Comment (NoelBright) [reply pb/block.proto]: done

Comment (dreamfly281) [reply pb/block.proto]: Suggest run go format before commit each time

Comment (NoelBright) [reply pb/block.proto]: this is .proto file. not a .go file...

**Pull request merged**